tests/scripts/pulp_deb/test_deb_sync_publish.sh

```diff
@@ -25,11 +25,19 @@ expect_succ pulp deb remote create \
   --distribution "$DEB_DISTRIBUTION"
 
 expect_succ pulp deb repository create \
+  --name "${ENTITIES_NAME}_repo" \
+  --remote "${ENTITIES_NAME}_remote"
+
+expect_succ pulp deb repository sync \
   --name "${ENTITIES_NAME}_repo"
 
 expect_succ pulp deb repository sync \
   --name "${ENTITIES_NAME}_repo" \
-  --remote "${ENTITIES_NAME}_remote"
+  --optimize
+
+expect_succ pulp deb repository sync \
+  --name "${ENTITIES_NAME}_repo" \
+  --no-optimize
```

Comment:
You need to guard these lines with something like
`if pulp debug has-plugin --name deb --min-version 2.20; then`

Reply:
Thanks for the hint.

Reply:
The extra syncs for the option are now conditional on the plugin version.